tests/endtoend/test_third_party_http_functions.py

```diff
@@ -35,22 +35,21 @@ def setUpClass(cls):
             host_json = cls.get_script_dir() / 'host.json'
             with open(host_json, 'w+') as f:
                 f.write(HOST_JSON_TEMPLATE)
-            os_environ = os.environ.copy()
-            # Turn on feature flag
-            os_environ['AzureWebJobsFeatureFlags'] = 'EnableWorkerIndexing'
```

Comment:
Where are we setting `EnableWorkerIndexing` now?

Reply:
In testutils and testutils_docker.py